Dockerfiles/ccp-api-server/Dockerfile

```diff
@@ -17,10 +17,14 @@ ENV PYTHONPATH="/opt/ccp_api_server"
 
 COPY . /opt/ccp_api_server
 
+RUN useradd 1001 && usermod -aG 0 1001 && chown 1001:0 /opt/ccp_api_server && chmod -R g+x /opt/ccp_api_server
```

Comment:
`chown -R 1001:0 /opt/ccp_api_server`

We need to change the permissions to root and all subsequent dirs recursively in that repo, WDYT ?
@mohammedzee1000

Reply:
This should not be a problem, but to be safe, yea, i guess it makes sense